### ISSUE TITLE
Lowercased names throw false warning for several namingConventions

### ISSUE THREAD

**Issue by awayken** [rossPatton/stylint#50]
Stylint currently throws a warning if your name is simply lowercased. I've seen this issue whether you use `camelCase`, `lowercase_underscore` or `lowercase-dash` as your namingConvention.

shorties.styl
```stylus
$color = black

.select {
    border: 1px solid $color;
}
```

shorties.json
```json
{
    "alphabetical": true,
    "borderNone": true,
    "brackets": false,
    "colons": false,
    "colors": false,
    "commaSpace": true,
    "commentSpace": false,
    "cssLiteral": false,
    "depthLimit": false,
    "duplicates": false,
    "efficient": true,
    "enforceVarStyle": false,
    "enforceBlockStyle": false,
    "extendPref": false,
    "globalDupe": false,
    "indentSpaces": 4,
    "leadingZero": true,
    "maxWarnings": 10,
    "maxWarningsKill": false,
    "mixed": false,
    "namingConvention": "lowercase-dash",
    "parenSpace": false,
    "placeholders": true,
    "semicolons": false,
    "Whitespace": true,
    "universal": true,
    "valid": false,
    "zeroUnits": true,
    "zIndexDuplicates": false,
    "zIndexNormalize": false
}
```

Output:
```bash
Warning:  preferred naming convention is lowercase-dash
File: ../temp-stylus/shorties.styl
Line: 1: $color = black

Warning:  preferred naming convention is lowercase-dash
File: ../temp-stylus/shorties.styl
Line: 3: .select {
```

Notice that the error throws on the variable `$color` and the class in the selector. I know namingConvention enforces rules on both.

**Comment by rossPatton** [comment]
fix incoming

**Comment by rossPatton** [comment]
alright, @awayken could you please pull the latest and double check that my fix is working as intended?

**Comment by awayken** [comment]
It looks like this is fixed in my latest pull. Short names worked for all four naming conventions. Thanks!
